### PR TITLE
Update sinatra to 2.2.1 to match rack-protection in core

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
```
/usr/bin/foreman-ruby: No such file or directory -- /usr/bin/bundle (LoadError)
```

Chicken, eggs, omelette!

File: plugins/ruby-foreman-tasks/debian/control [hunk 1]
```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, git | git-core, foreman (>= 2.6.0~rc1), foreman-nulldb (>= 2.1.0~rc1), foreman-assets, python-dev
+Build-Depends: debhelper, git | git-core, foreman (>= 2.6.0~rc1), foreman-nulldb (>= 2.1.0~rc1), foreman-assets, python-dev, bundler
```
Comment (evgeni) [inline plugins/ruby-foreman-tasks/debian/control:5]: That's a cheap workaround for now, will be dropped in the big "plugin.mk" change again

**Review by ekohl — COMMENTED** [review]
Does it make sense to combine this with https://github.com/theforeman/foreman-packaging/pull/8234 or do you want to cherry pick this?

**Comment by evgeni** [comment]
> Does it make sense to combine this with #8234 or do you want to cherry pick this?

I want/need to pick this to 3.2 and 3.3 once we rebuild that and have the new rack-protection there too.

**Review by ekohl — APPROVED** [review]

**Pull request merged**